lib/swoosh/adapters/postmark.ex

```diff
@@ -8,8 +8,9 @@ defmodule Swoosh.Adapters.Postmark do
 
       # config/config.exs
       config :sample, Sample.Mailer,
-        adapter: Swoosh.Adapters.Postmark,
-        api_key: "my-api-key"
+        adapter:  Swoosh.Adapters.Postmark,
+        api_key:  "my-api-key",
+        template: true # optional
```

Comment:
The construction of the endpoint happens when you call `deliver/2`, so it happens every time you send an email.

I really believe this should be attached to the email rather than a config on the adapter.